src/groupeddataframe/groupeddataframe.jl

```diff
@@ -245,6 +245,10 @@ equivalent `Tuple` or `NamedTuple`.
 Instances of this type are returned by `keys(::GroupedDataFrame)` and are not
 meant to be constructed directly.
 
+Indexing is one-dimensional like specifying a column of a `DataFrame`.
```

Comment:
I don't understand the "like" part.

Reply:
I have copied it from `DataFrameRow` docstring, so we should fix it also there. It is meant that the rules are the same as for the second dimension in data frame. How do you think we could phrase it?

Reply:
OK. Maybe something about accepted index types?